src/AudioUnit/AUParameter.cs

```diff
@@ -17,7 +19,7 @@ public partial class AUParameter
 		public string GetString (float? value)
 		{
 			unsafe {
-				if (value != null && value.HasValue) {
+				if (value is not null && value.HasValue) {
```

Comment:
NBD, but this is doing the same thing twice, but in two different ways. This can be simplified as:

```suggestion
				if (value is not null) {
```